Adyen.xcodeproj/project.pbxproj

```diff
@@ -4640,7 +4640,7 @@
 				E9021DFC2260D626009CF7F4 /* ComponentSections.swift in Sources */,
 				E7484C2723B9FDF000CCF74F /* SlideInPresentationAnimator.swift in Sources */,
 				5A238C93263AEA0800758FEF /* PreApplePayView.swift in Sources */,
-				E9B36CC722451C8300EAA368 /* PaymentMethodsConfiguration.swift in Sources */,
+				E9B36CC722451C8300EAA368 /* Configuration.swift in Sources */,
```

Comment:
lets call this file `DropInConfiguration.swift` for better discoverability, since we have already a file called `Configuration.swift`

Reply:
👍